packages/ibm-products/src/components/CreateFullPage/CreateFullPageStep.tsx

```diff
@@ -28,6 +30,90 @@ const defaults = {
   includeStep: true,
 };
 
+interface CreateFullPageStepBaseProps {
+  /**
+   * Content that shows in the CreateFullPage step
+   */
+  children?: ReactNode;
```

Comment:
You can extend the prop interface with `PropsWithChildren` interface, instead of this.
`interface CreateFullPageStepBaseProps extends PropsWithChildren {`